lib/spec/openapi/utils.js

```diff
@@ -99,8 +99,24 @@ function plainJsonObjectToOpenapi3 (container, jsonSchema, externalSchemas) {
         const result = {
           in: container,
           name: propertyName,
-          required: jsonSchemaElement.required,
-          schema: jsonSchemaElement
+          required: jsonSchemaElement.required
+        }
+        // complex serialization in query or cookie, eg. JSON
+        // https://swagger.io/docs/specification/describing-parameters/#schema-vs-content
+        if (jsonSchemaElement.consume) {
```

Comment:
as other PRs, it could be more friendly to adopt a name like `x-consume` to warn the uses that, that field is not standard

Reply:
Ready. "x-consume" with "x-response-description" moved to constants